Polyhedron/demo/Polyhedron/Scene_polylines_item.cpp

```diff
@@ -17,6 +17,16 @@ struct Scene_polylines_item_private {
         spheres_drawn_radius(0)
     {
       item = parent;
+      invalidate_stats();
+    }
+    void invalidate_stats()
+    {
+      nb_vertices = 0;
+      nb_edges = 0;
+      min_length = INT_MAX;
```

Comment:
`INT_MAX` for a double?! It should be [`std::numeric_limits<double>`][l]`::`[`max()`][m].

[l]: http://en.cppreference.com/w/cpp/types/numeric_limits
[m]: http://en.cppreference.com/w/cpp/types/numeric_limits/max